containers/grizzly2-http/pom.xml

```diff
@@ -61,19 +61,6 @@
                 <inherited>true</inherited>
             </plugin>
         </plugins>
-
-        <resources>
-            <resource>
-                <directory>${basedir}/src/main/java</directory>
```

Comment:
Why do you have META-INF in src/main/java? I would move it to src/main/resources.

Reply:
this whole block is redundant and is being removed from Jersey...

Reply:
Yes, but still, things like META-INF shall be in `src/main/resources`.